website/content/installation/clouds.md

```diff
@@ -90,15 +90,16 @@ After that, MetalLB should work normally.
 ### MetalLB on OpenStack
 
 You can run a Kubernetes cluster on OpenStack VMs, and use MetalLB as
-the load balancer. However you have to disable OpenStack's ARP
-spoofing protection if you want to use L2 mode. You must disable it on
-all the VMs that are running Kubernetes.
-
-By design, MetalLB's L2 mode looks like an ARP spoofing attempt to
-OpenStack, because we're announcing IP addresses that OpenStack
-doesn't know about. There's currently no way to make OpenStack
-cooperate with MetalLB here, so we have to turn off the spoofing
-protection entirely.
+the load balancer. By design, MetalLB's L2 mode looks like an ARP
+spoofing attempt to OpenStack, because we're announcing IP addresses
+that OpenStack doesn't know about. 
+
+You can  disable OpenStack's ARP spoofing protection if you want to
```

Comment:
```suggestion
You can disable OpenStack's ARP spoofing protection if you want to
```

Reply:
to disable spoofing protection, you can mention that port security needs to be disabled on the Neutron port.